django/core/checks/model_checks.py

```diff
@@ -155,3 +156,31 @@ def build_error(model_key, func, args, keywords):
 @register(Tags.models)
 def check_lazy_references(app_configs=None, **kwargs):
     return _check_lazy_references(apps)
+
+
+@register(Tags.models)
```

Comment:
The check should probably go in `django/db/models/base.py`. As registered, this isn't called by `Model.check()`.